<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>All_Bill_Dialog</class>
 <widget class="QDialog" name="All_Bill_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1296</width>
    <height>622</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bill</string>
  </property>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>-40</y>
     <width>20</width>
     <height>631</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_11">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>20</y>
     <width>241</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>OPD</string>
   </property>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>220</width>
      <height>70</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>220</width>
      <height>70</height>
     </size>
    </property>
    <property name="title">
     <string>QTY:</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>200</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>30</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>220</width>
      <height>70</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>220</width>
      <height>70</height>
     </size>
    </property>
    <property name="title">
     <string>RATE:</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>200</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>30</height>
      </size>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_12">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>30</y>
     <width>241</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>IPD</string>
   </property>
   <widget class="QGroupBox" name="groupBox_15">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>220</width>
      <height>70</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>220</width>
      <height>70</height>
     </size>
    </property>
    <property name="title">
     <string>QTY:</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>200</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>30</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_16">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>220</width>
      <height>70</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>220</width>
      <height>70</height>
     </size>
    </property>
    <property name="title">
     <string>RATE:</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>200</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>30</height>
      </size>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_17">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>200</y>
     <width>241</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>PNO</string>
   </property>
   <widget class="QGroupBox" name="groupBox_18">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>220</width>
      <height>70</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>220</width>
      <height>70</height>
     </size>
    </property>
    <property name="title">
     <string>QTY:</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>200</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>30</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_19">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>220</width>
      <height>70</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>220</width>
      <height>70</height>
     </size>
    </property>
    <property name="title">
     <string>RATE:</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_12">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>200</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>30</height>
      </size>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_20">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>210</y>
     <width>241</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>SPEECH THERAPY</string>
   </property>
   <widget class="QGroupBox" name="groupBox_23">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>220</width>
      <height>70</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>220</width>
      <height>70</height>
     </size>
    </property>
    <property name="title">
     <string>QTY:</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_15">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>200</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>30</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_24">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>220</width>
      <height>70</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>220</width>
      <height>70</height>
     </size>
    </property>
    <property name="title">
     <string>RATE:</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_16">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>200</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>30</height>
      </size>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_25">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>30</y>
     <width>241</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>ACCUPUNTURE</string>
   </property>
   <widget class="QGroupBox" name="groupBox_26">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>220</width>
      <height>70</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>220</width>
      <height>70</height>
     </size>
    </property>
    <property name="title">
     <string>QTY:</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_17">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>200</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>30</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_27">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>220</width>
      <height>70</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>220</width>
      <height>70</height>
     </size>
    </property>
    <property name="title">
     <string>RATE:</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_18">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>200</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>30</height>
      </size>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_28">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>30</y>
     <width>251</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>DRESSING</string>
   </property>
   <widget class="QGroupBox" name="groupBox_29">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>220</width>
      <height>70</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>220</width>
      <height>70</height>
     </size>
    </property>
    <property name="title">
     <string>QTY:</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_19">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>200</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>30</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_30">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>220</width>
      <height>70</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>220</width>
      <height>70</height>
     </size>
    </property>
    <property name="title">
     <string>RATE:</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_20">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>200</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>30</height>
      </size>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_31">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>210</y>
     <width>241</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>ROOM</string>
   </property>
   <widget class="QGroupBox" name="groupBox_32">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>220</width>
      <height>70</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>220</width>
      <height>70</height>
     </size>
    </property>
    <property name="title">
     <string>QTY:</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_21">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>200</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>30</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_33">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>220</width>
      <height>70</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>220</width>
      <height>70</height>
     </size>
    </property>
    <property name="title">
     <string>RATE:</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_22">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>200</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>30</height>
      </size>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_34">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>220</y>
     <width>241</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>PACKAGE</string>
   </property>
   <widget class="QGroupBox" name="groupBox_35">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>220</width>
      <height>70</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>220</width>
      <height>70</height>
     </size>
    </property>
    <property name="title">
     <string>QTY:</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_23">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>200</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>30</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_36">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>220</width>
      <height>70</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>220</width>
      <height>70</height>
     </size>
    </property>
    <property name="title">
     <string>RATE:</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_24">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>200</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>30</height>
      </size>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_37">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>390</y>
     <width>241</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>OTHERS</string>
   </property>
   <widget class="QGroupBox" name="groupBox_38">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>220</width>
      <height>70</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>220</width>
      <height>70</height>
     </size>
    </property>
    <property name="title">
     <string>QTY:</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_25">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>200</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>30</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_39">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>220</width>
      <height>70</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>220</width>
      <height>70</height>
     </size>
    </property>
    <property name="title">
     <string>RATE:</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_26">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>200</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>30</height>
      </size>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>22</y>
     <width>220</width>
     <height>70</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>220</width>
     <height>70</height>
    </size>
   </property>
   <property name="title">
    <string>Patient's Name:</string>
   </property>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>30</height>
     </size>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>98</y>
     <width>250</width>
     <height>70</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>220</width>
     <height>70</height>
    </size>
   </property>
   <property name="title">
    <string>Bill No:</string>
   </property>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>200</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>30</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>172</y>
     <width>220</width>
     <height>70</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>220</width>
     <height>70</height>
    </size>
   </property>
   <property name="title">
    <string>Discount:</string>
   </property>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>200</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>30</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>324</y>
     <width>220</width>
     <height>70</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>220</width>
     <height>70</height>
    </size>
   </property>
   <property name="title">
    <string>Date:</string>
   </property>
   <widget class="QDateEdit" name="dateEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>200</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>30</height>
     </size>
    </property>
    <property name="displayFormat">
     <string>dd/MM/yyyy</string>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>248</y>
     <width>220</width>
     <height>70</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>220</width>
     <height>70</height>
    </size>
   </property>
   <property name="title">
    <string>Extra:</string>
   </property>
   <widget class="QLineEdit" name="lineEdit_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>200</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>30</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>221</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string>Proceed</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>450</y>
     <width>250</width>
     <height>147</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>250</width>
     <height>70</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 36pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="title">
    <string>Total</string>
   </property>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>230</width>
      <height>61</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>230</width>
      <height>30</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 28pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_9">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>400</y>
     <width>250</width>
     <height>147</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>250</width>
     <height>70</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 36pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="title">
    <string>ADVANCE:</string>
   </property>
   <widget class="QLineEdit" name="lineEdit_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>231</width>
      <height>61</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>30</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_10">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>400</y>
     <width>250</width>
     <height>147</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>250</width>
     <height>70</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 36pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="title">
    <string>REMAIN:</string>
   </property>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>231</width>
      <height>71</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>230</width>
      <height>30</height>
     </size>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>560</y>
     <width>230</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>230</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string>Finish</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>comboBox</tabstop>
  <tabstop>lineEdit</tabstop>
  <tabstop>lineEdit_4</tabstop>
  <tabstop>lineEdit_5</tabstop>
  <tabstop>dateEdit</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>lineEdit_2</tabstop>
  <tabstop>lineEdit_3</tabstop>
  <tabstop>lineEdit_10</tabstop>
  <tabstop>lineEdit_9</tabstop>
  <tabstop>lineEdit_18</tabstop>
  <tabstop>lineEdit_17</tabstop>
  <tabstop>lineEdit_20</tabstop>
  <tabstop>lineEdit_19</tabstop>
  <tabstop>lineEdit_12</tabstop>
  <tabstop>lineEdit_11</tabstop>
  <tabstop>lineEdit_16</tabstop>
  <tabstop>lineEdit_15</tabstop>
  <tabstop>lineEdit_22</tabstop>
  <tabstop>lineEdit_21</tabstop>
  <tabstop>lineEdit_24</tabstop>
  <tabstop>lineEdit_23</tabstop>
  <tabstop>lineEdit_26</tabstop>
  <tabstop>lineEdit_25</tabstop>
  <tabstop>lineEdit_6</tabstop>
  <tabstop>pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
